src/components/ModalAddressQRCode.js

```diff
@@ -83,7 +89,7 @@ class ModalAddressQRCode extends React.Component {
           <div className="modal-content">
             <div className="modal-header">
               <h5 className="modal-title" id="exampleModalLabel">{t`Address to receive tokens`}</h5>
-              <button type="button" className="close" data-dismiss="modal" aria-label="Close">
+              <button type="button" className="close" onClick={this.close} data-dismiss="modal" aria-label="Close">
```

Comment:
It should be `this.props.onClose`, fixed in 708d6e29ab5bda9ac7800232fd669c2762351ef5

It was working though, because of the event and `data-dismiss`